use-case-guides/salesforce-account-lookup/README.md

```diff
@@ -0,0 +1,161 @@
+- Salesforce Connector built in Creator Studio (follow [our guide](https://developer.moveworks.com/creator-studio/resources/authentication-guide?id=salesforce))
+
+# What are we building?
+
+## Conversation Design
+
+[This purple chat](https://developer.moveworks.com/creator-studio/developer-tools/purple-chat-builder/?workspace=%7B%22title%22%3A%22My+Workspace%22%2C%22botSettings%22%3A%7B%7D%2C%22mocks%22%3A%5B%7B%22id%22%3A8491%2C%22title%22%3A%22Mock+1%22%2C%22transcript%22%3A%7B%22settings%22%3A%7B%22colorStyle%22%3A%22LIGHT%22%2C%22startTime%22%3A%2211%3A43+AM%22%2C%22defaultPerson%22%3A%22GWEN%22%2C%22editable%22%3Atrue%7D%2C%22messages%22%3A%5B%7B%22from%22%3A%22USER%22%2C%22text%22%3A%22What+is+the+Renewal+date+for+ACME%3F%22%7D%2C%7B%22from%22%3A%22ANNOTATION%22%2C%22text%22%3A%22Query+Salesforce+Accounts+API+endpoint%22%7D%2C%7B%22from%22%3A%22BOT%22%2C%22text%22%3A%22The+renewal+date+for+ACME+is+on+February+25%2C+2026%22%7D%2C%7B%22from%22%3A%22USER%22%2C%22text%22%3A%22Perfect%2C+thanks%21%22%7D%5D%7D%7D%5D%7D) shows the experience we are going to build.
+
+## Creator Studio Components
+
+- **Triggers:**
+    1. Natural Language
+- **Slots**:
+    1. Name of the Salesforce Account
+- **Actions:**
+    1. Query Customer Account details
+- **Guidelines:**
+    1. None
+
+Based on the needs of this use case, we should build a [Lookup Single Record by ID / Keyword](https://developer.moveworks.com/creator-studio/design-patterns/dp-6/)**.**
+
+## API Research
+
+There’s only 1 API needed to build this use case
+
+### API #1: Account Lookups
+
+Since we want to integrate with Salesforce, we should look into their SOQL Query APIs. For this use case, we will need only 1 API call.
+
+![Untitled](Use%20Case%20Tutorial%20Salesforce%20Account%20Lookup%200b5238e472eb4113bbc51e0a65fa3e08/Untitled.png)
+
+1. [Salesforce SOQL Query API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/resources_query.htm): An API for Salesforce to execute the specified SOQL Query. Here, our query would be related to the `Account` object
+    1. Sample SOQL Query API: Here is an example of a sample query API using a SOQL query on the `Account` object.
+
+        ```bash
+        curl --location --globoff 'https://{{your-salesforce-instance}}.my.salesforce.com/services/data/v58.0/query?q=SELECT%20Name%2CType%2CDescription%2CId%2CWebsite%2COwner.Name%20FROM%20Account%20WHERE%20Name%20LIKE%20%27{{query}}%27' \
+        --header 'Authorization: Bearer {{generated_bearer_token}}'
+        ```
+
+        The Query we used here is:
+
+        ```sql
+        SELECT
+         Name,
+         Type,
+         Description,
+         Id,
+         Website,
+         Owner.Name
+        FROM
+         Account
+        WHERE
+         Name LIKE '{{query}}'
+        ```
+
+        Use this Query to fetch all relevant details from your customer’s Salesforce account by replacing the `{{query}}` field with the name of the customer you’re trying to lookup for.
```

Comment:
```suggestion
        Use this Query to fetch all relevant details from your customer’s Salesforce account by replacing the `{{query}}` field with the name of the customer you’re trying to lookup for. The `%25` searches for the Account inside account names, instead of an exact keyword match (Example: matching `Acme` with "Acme Inc." and "Acme Semiconductors", instead of matching no results.
```